### PR TITLE
Added update method to PatientApi, and added patient to CapabilityStatement

### PR DESCRIPTION
**Pull request by that-guy-is-it**

### CODE REVIEW

File: patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/patient/PatientApi.java [hunk 1]
```diff
@@ -51,4 +52,35 @@ Patient patientRead(
...
                       + " assigned by the Master Veteran Index (MVI).")
           String id);
+
+  @Operation(
+      summary = "Patient Update",
+      description =
+          "https://build.fhir.org/ig/HL7/US-Core-R4/StructureDefinition-us-core-patient.html",
+      tags = {"Patient"})
+  @PUT
+  @Path("Patient/c{id}")
```
Comment (apeirats-va) [inline patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/patient/PatientApi.java:62]: typo?

Comment (that-guy-is-it) [reply patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/patient/PatientApi.java:62]: Oh dear, yes.

File: patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/patient/PatientApi.java [hunk 2]
```diff
@@ -51,4 +52,35 @@ Patient patientRead(
...
+  @ApiResponse(
+      responseCode = "200",
+      description = "Record found",
```
Comment (apeirats-va) [inline patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/patient/PatientApi.java:65]: updated*

File: patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/patient/PatientApi.java [hunk 3]
```diff
@@ -51,4 +52,35 @@ Patient patientRead(
...
+              mediaType = "application/fhir+json",
+              schema = @Schema(implementation = OperationOutcome.class)))
+  Patient patientUpdate(
+      @Parameter(
+              in = ParameterIn.PATH,
+              name = "id",
+              required = true,
+              description =
+                  "The logical id of the resource." + " Once assigned, this value never changes.")
+          String id);
```
Comment (apeirats-va) [inline patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/patient/PatientApi.java:85]: Add `@RequestBody`. See Questionnaire for example

**Comment by jblefkove-va** [comment]
LGTM

**Review by apeirats-va — APPROVED** [review]

**Pull request merged**